scripts/nf1_classifier.py

```diff
@@ -361,8 +366,10 @@ def plot_decision_function(score_df, partition, output_file):
                             y_score = clf.decision_function(x_test)
                             y_score_train = clf.decision_function(x_train)
 
-                            fpr, tpr, _ = roc_curve(y_test, y_score)
-                            fpr_t, tpr_t, _ = roc_curve(y_train, y_score_train)
+                            fpr, tpr, _ = roc_curve(y_test, y_score,
```

Comment:
Not as simple as this makes it seem! After many hours of interpolating over the ROC lines and trying many different potential solutions I have determined that this problem does not work as well for these particular points as it does in the example!

I am going to go with the [line graph here](https://github.com/greenelab/nf1_inactivation/pull/10#issuecomment-263662730) because it does tell the whole story of the figure and fits with the audience of the journal.

At some point in the future if you come across a similar scenario, or want to work through this one with me, that would be great.